statvar_imports/india_ndap/ndap/download_script.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import pandas as pd
+import config
+import os
+from absl import logging, app
+
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../../util/')) 
+from download_util_script import _retry_method
+
+
+def main(_):
+    all_data = []
+
+    for page_num in range(1, 3):  
+        api_url = f"{config.url}&pageno={page_num}"
```

Comment:
Why is the page numbers fixed to 2 pages ? Can we dynamically get the number of pages from response to iterate over ?